ruddock/templates/info.html

```diff
@@ -0,0 +1,31 @@
+{% extends "layout.html" %}
+{% block body %}
+<h2>Useful House Info</h2>
+<h3>Door codes</h3>
+<table>
+<tr><td>Computer lab</td><td>34-5-2-1</td></tr>
+<tr><td>PGR</td><td>1-34-2</td></tr>
+<tr><td>O'Doms</td><td>5-3-2-4</td></tr>
+<tr><td>Kitchens</td><td>24-3-1-5</td></tr>
+</table>
```

Comment:
Uh, not sure what happened. That was supposed to say "if you added an extra `<br>` after each table". Newlines in the html don't do anything.

It seems like Github interpreted the `<br>` as part of the actual html? But it doesn't seem like it's susceptible to the most obvious xss attacks, since adding `<script>...</script>` tags don't do anything. Weird.